java/clients/src/main/java/sleeper/clients/util/table/TableWriterPropertyHelper.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2022-2025 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.clients.util.table;
+
+import sleeper.clients.util.table.TableRow.Builder;
+import sleeper.core.properties.SleeperProperty;
+import sleeper.core.properties.table.TableProperty;
+
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+public class TableWriterPropertyHelper {
+
+    private TableWriterPropertyHelper() {
+    }
+
+    public static TableWriter generateTableBuildForGroup(Stream<SleeperProperty> propertyStream) {
```

Comment:
It's a bit difficult to tell which of these methods are the entrypoint for this class and how it will be used.

Can we make the other methods private, and use a method signature where we pass in an OutputStream rather than return a TableWriter?